### PR TITLE
measure: add json output

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

**Comment by bluca** [comment]
Can you add a test for the output in TEST-70? Just to ensure we don't somehow break it later on, given it would be an api-like output

**Comment by poettering** [comment]
There's a test for this included in #24351, hence merging already.

File: src/boot/measure.c [hunk 1]
```diff
@@ -47,15 +50,18 @@ static int help(int argc, char *argv[], void *userdata) {
                "  status             Show current PCR values\n"
                "  calculate          Calculate expected PCR values\n"
```
Comment (yuwata) [inline src/boot/measure.c:51]: Maybe better to align descriptions for verbs?

Comment (poettering) [reply src/boot/measure.c:51]: Actually, the #24351 PR adds more stuff to --help which makes this aligned properly again.

**Pull request merged**